examples/offline_inference/eagle.py

```diff
@@ -48,7 +48,8 @@ def main():
     args = parser.parse_args()
 
     model_dir = "meta-llama/Meta-Llama-3-8B-Instruct"
-    eagle_dir = "abhigoyal/EAGLE-LLaMA3-Instruct-8B-vllm"
+    # eagle_dir = "yuhuili/EAGLE-LLaMA3-Instruct-8B"
+    eagle_dir = "lmsys/sglang-EAGLE-LLaMA3-Instruct-8B"
```

Comment:
using sglang model so that the prev SGL bench is comparable: https://docs.google.com/document/d/18ETJLsnxR88Qq3VDk5Mq-Hb7vuE9o3VNZ-hhz-OqAXk/edit?usp=sharing